chatcommunicate.py

```diff
@@ -119,18 +132,27 @@ def watcher(ev, wrap2):
             current_message = shortcut_messages[i]
             if length > 1:
                 reply += str(i + 1) + ". "
-            reply += "[" + current_message.split(" ")[0] + "] "
+            reply += "[{0}] ".format(current_message.split(" ")[0])
             if current_message.split(" ")[1] != "-":
-                r = handle_commands(current_message.lower(), current_message.split(" "), ev_room, ev_room_name, ev_user_id, ev_user_name, wrap2, current_message, message_id)
-                result = r
+                pure_result = handle_commands(
+                    current_message.lower(),
+                    current_message.split(" "),
+                    ev_room, ev_room_name,
+                    ev_user_id,
+                    ev_user_name,
+                    wrap2,
+                    current_message,
+                    message_id
+                )
+                result = pure_result
```

Comment:
I just renamed the variables there, in the following lines the properties of `result` get changed while `pure_result` stay pure from the source. I have no idea why the code is like that... I can take a look at refactoring it if you want @AWegnerGitHub